packages/oruga-next/src/components/input/Input.vue

```diff
@@ -490,6 +507,15 @@ const counterClasses = defineClasses(["counterClass", "o-input__counter"]);
             @invalid="onInvalid"
             @input="onInput" />
 
+        <o-icon
```

Comment:
Changing the HTML structure is a breaking change. Styling could be based on a specific HTML structure. Is it necessary to move the icon element here?

Reply:
Ahh ok, I get it. I changed it because I thought this html structure would be more intuitive. But that's fine with your argument.